app/controllers/cameras_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CamerasController < ApplicationController
+before_action :set_camera, only: [:index, :show, :edit, :update, :destroy]
+
+  def index
+    @cameras = Camera.all
+  end
+
+  def new
+    @camera = Camera.new
+  end
+
+  def create
+    @camera = Camera.new(camera_params)
+    if @camera.save
+      redirect_to cameras_show_path(@camera)
+    else
+      render :new
+    end
+  end
+
+  def show
+  end
+
+  def edit
+  end
+
+  def update
+    @camera.update(@camera_params)
+    redirect_to camera_show_path(@camera)
```

Comment:
redirect_to camera_path(@CAMERA)

*Ignore why camera is all uppercase. Not sure why github is doing that.